packages/documentation/pages/usage/components/form-fields.vue

```diff
@@ -307,17 +307,27 @@
 								label="actions"
 								type="switch"
 							/>
-							<KtFieldSingleSelect
+							<div
```

Comment:
Ideally, this probably should’ve been handled similar to the other "potentially supported" fields. E.g. this one:

```vue
<KtFieldNumber
	v-if="componentDefinition.supports.tabIndex"
	formKey="tabIndex"
	helpText="Support varies"
	isOptional
	label="tabIndex"
/>
```

Not a blocker for the PR, just would’ve also helped to avoid all the `additionalProps` entries in this diff